packages/vue/src/components/authenticator.vue

```diff
@@ -77,17 +64,13 @@ const emit = defineEmits([
   'verifyUserSubmit',
   'confirmVerifyUserSubmit',
 ]);
-const machine = createAuthenticatorMachine();
 
-const service = useInterpret(machine);
-let unsubscribeHub: () => void;
 let unsubscribeMachine: () => void;
 
-const { state, send } = useActor(service);
-useAuth(service);
-
 const hasInitialized = ref(false);
 
+const { service, send, state } = useAuth();
```

Comment:
I dumbed down the `useAuth`, and `state` and `send` are no longer used for rendering logic. We would ideally just remove `useAuth` in this file too, but we still rely it in some places below publicly, and I wanted to minimize big changes to this file.